instrumentation/armeria-0.99/auto/armeria-0.99-auto.gradle

```diff
@@ -0,0 +1,24 @@
+ext {
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+// TODO(anuraaga): Move into instrumentation.gradle
+archivesBaseName = projectDir.parentFile.name
+
+muzzle {
+  pass {
+    group = "com.linecorp.armeria"
+    module = "armeria"
+    versions = "[0.99.8,)"
```

Comment:
would it be weird to rename the module to armeria-1.0 now, just to avoid unnecessary maven artifact naming churn later? we can still doc the "correct" as of now version in README

Reply:
Yeah think that makes sense